data/free worlds intro.txt

```diff
@@ -546,6 +572,26 @@ mission "FW Katya 1"
 	on decline
 		event "FW Katya Alt 2: ready" 21
 
+mission "logbook: karate chop decline"
+	invisible
+	to offer
+		has "karate chop"
+		has "FW Katya 1: offered"
+	on offer
+		log `I karate chopped a militia officer who was holding a gun to the back of my head. Luckily the gun wasn't loaded, but the karate chop didn't work as expected. (I knew I should have studied karate while on New Boston...)`
+		fail
+
+mission "logbook: no karate chop decline"
+	invisible
+	to offer
+		not "karate chop"
+		has "FW Katya 1: offered"
+		not "FW Katya 1: active"
+		not "FW Katya 1: done"
+	on offer
+		log `A militia officer tried to recruit me to help the Free Worlds by holding a gun to the back of my head! The Navy better do something about this before someone gets hurt.`
```

Comment:
`The Navy better do something about this before someone gets hurt.`

The start of this feels weird.